GitUI/CommandsDialogs/RefLogDialog/BranchNameForm.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Windows.Forms;
+
+namespace GitUI.CommandsDialogs.RefLogDialog
+{
+    public partial class BranchNameForm : Form
```

Comment:
Branch name should be normalized according to the settings. See the IGitBranchNameNormaliser and its usages for details.